packages/magento-product/components/AddProductsToCart/AddProductsToCartSnackbar.tsx

```diff
@@ -69,7 +76,7 @@ export function AddProductsToCartSnackbar(props: AddProductsToCartSnackbarProps)
           <Trans
             id='<0>{name}</0> has been added to your shopping cart!'
             components={{ 0: <strong /> }}
-            values={{ name: items[items.length - 1]?.product.name }}
+            values={{ name: productsAdded.join(', ') }}
```

Comment:
@JoshuaS98 Minor optimisation, but maybe use [Intl.ListFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat) for this?

We should also probably render a different sentence based on it being one or more products.
For a single product "has been added" is fine. For multiple products it should be "have been added".

Reply:
Maybe the simple solution would be to do something like: 'Products have been added to your shopping cart!'. It doesn't happen too often though.